ui/src/AppCharting/AppChartC3.svelte

```diff
@@ -291,6 +291,8 @@
     }, 0);
   }
 
+  $: console.log('DATA', chartData);
```

Comment:
Still need this?

Reply:
I do not, thanks good catch. I'll remove these in my next commit